lab_3_keywords_textrank/main.py

```diff
@@ -564,7 +671,8 @@ def get_top_keywords(self, n_keywords: int) -> tuple[int, ...]:
             tuple[int, ...]
                 top n most important tokens in the encoded text
         """
-        pass
+        srtd_by_key = sorted(self._scores)
```

Comment:
Although we did discuss that dictionaries are ordered, I still feel that this is a strange practice...

Reply:
I've discussed it with senior colleagues, we've come to the conclusion that to establish secondary sorting order it is better to use `key` argument of the `sorted` function, as it _is_ possible to pass two sorting keys there

Reply:
done